code/game/gamemodes/objective.dm

```diff
@@ -768,5 +768,5 @@ var/list/potential_theft_objectives = subtypesof(/datum/theft_objective) \
 //wizard
 
 /datum/objective/wizchaos
-	explanation_text = "Wreak havoc upon the station as much you can. Send those wandless Nanotrasen scum a message!"
+	explanation_text = "Wreak havoc upon the station as much you can. Send those wandless NanoTrasen scum a message!"

```

Comment:
As @Regen1 says, I'm unaware of that. If thats the case, then other places such as the wiki need to be updated.

If you could please confirm @Necaladun @FreeStylaLT on NanoTrasen vs Nanotrasen, I'll update this PR, and make sure the wiki falls in line too.